packages/ripple-binary-codec/src/types/uint-64.ts

```diff
@@ -1,11 +1,9 @@
 import { UInt } from './uint'
 import { BinaryParser } from '../serdes/binary-parser'
-import bigInt = require('big-integer')
-import { isInstance } from 'big-integer'
 import { Buffer } from 'buffer/'
 
 const HEX_REGEX = /^[a-fA-F0-9]{1,16}$/
-const mask = bigInt(0x00000000ffffffff)
+const mask = BigInt(0x00000000ffffffff)
```

Comment:
There's actually a second way of assigning a BigInt literal by appending `n` to the end of the literal. For example, `0x00000000ffffffffn` instead of using the constructor `BigInt(0x00000000ffffffff)`. I do like the constructor way because it's more apparent it's a BigInt versus appending an `n` at the end of the literal where it's more subtle. Not sure if there are type conversion implications with using one over the other.

Reply:
With hex it for sure could be confusing to add the n.